mmdet3d/datasets/nuscenes_mono_dataset.py

```diff
@@ -76,16 +77,55 @@ class NuScenesMonoDataset(CocoDataset):
 
     def __init__(self,
                  data_root,
+                 ann_file,
+                 pipeline,
                  load_interval=1,
                  with_velocity=True,
                  modality=None,
                  box_type_3d='Camera',
                  eval_version='detection_cvpr_2019',
                  use_valid_flag=False,
                  version='v1.0-trainval',
-                 **kwargs):
-        super().__init__(**kwargs)
+                 classes=None,
+                 img_prefix='',
+                 seg_prefix=None,
+                 proposal_file=None,
+                 test_mode=False,
+                 filter_empty_gt=True,
+                 file_client_args=dict(backend='disk')):
```

Comment:
We made these changes to make the init args more visible and clear in this dataset class?

Reply:
We need the args for `self.pipeline = Compose(pipeline)` using mmdet3d's `Compose` , instead of original mmdet `Compose` .